index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Mobile Version Skeletun</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav>
+    <ul class="bar">
+      <li>John Doe</li>
+      <li><img src="Union.png" alt="image of three rows"></li>
+    </ul>
+  </nav>
+  <section class="main">
+    <div class="container">
+      <h1>Hey There<br>I'm Alex</h1>
+      <h2>I'm a Software Developer</h2>
+      <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </div>
+    <div class="icons">
+      <img src="cat.png" alt="github icon">
+      <img src="Linkedin.png" alt="linkedin icon">
+      <img src="hand.png" alt="angelist icon">
+      <img src="bird.png" alt="twitter icon">
+      <img src="m.png" alt="medium icon">
+    </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) as per the project requirements